ruby/Fibonacci.rb

```diff
@@ -0,0 +1,53 @@
+
+  def fibonacci2
+    MAX_VALUE = 4000000
+
+    (1..MAX_VALUE).each_with_object([1,2]) do |num, acum|
+      return acum if acum[acum.size - 1] + acum[acum.size - 2] > MAX_VALUE 
```

Comment:
we can assign this to a variable acum[acum.size - 1] + acum[acum.size - 2]. because is being used in two parts. here and below.